### DISCUSSION TITLE
new html content is coming from server side but not getting updated in frontend

### DISCUSSION THREAD

**Discussion by parmonov98** [livewire/livewire#3891] [7. Bugs (archived)]
there is a navigation in sidebar which consists of company, project, teams and employees
pro30(new project name) should be added to tree-view after calling 
`$this->emitTo('tree-view', 'rerender');`
I checked that in TreeView component's rerender method, ping goes there. I mean the rerender event triggered in TreeView component.
 
What I should get is **pro30**  added to list
![image](https://user-images.githubusercontent.com/24564577/135064561-69e16bf0-552c-46e1-8d83-54e47ee207a7.png)

what got in response from http://webdashboard.test/livewire/message/tree-view 
below and I see that pro30 item in response but not being added to frontend
`
{"effects":{"html":"<b wire:id=\"VMaH8tDMMGwCG3EH7YNk\">Navigation in company<\/b>\n<ul id=\"myUL\" >\n                        <li>\n                <span class=\"caret caret-down\">second company (<b>18<\/b>)<\/span>\n                <ul class=\"nested active\">\n                    \n                                                    <li>pro33<\/li>\n                                            \n                                                    <li>\n                                <span class=\"caret caret-down\">project 2 (<b>3<\/b>)<\/span>\n                                <ul class=\"nested active\">\n                                                                                                                        <li>\n\n                                                <span class=\"caret caret-down\">team for 2 (<b>1<\/b>)<\/span>\n                                                <ul class=\"nested active\">\n                                                    \n                                                        <li>new  employee2<\/li>\n                                                                                                    <\/ul>\n                                            <\/li>\n                                                                                                                                                                <li>\n\n                                                <span class=\"caret caret-down\">team 2 (<b>1<\/b>)<\/span>\n                                                <ul class=\"nested active\">\n                                                    \n                                                        <li>Murod2 Parmonov2<\/li>\n                                                                                                    <\/ul>\n                                            <\/li>\n                                                                                                                                                                <li>\n\n                                                <span class=\"caret caret-down\">kjbbkjbjk (<b>0<\/b>)<\/span>\n                                                <ul class=\"nested active\">\n                                                    \n                                                                                                    <\/ul>\n                                            <\/li>\n                                                                                                            <\/ul>\n                            <\/li>\n                                            \n                                                    <li>project 3<\/li>\n                                            \n                                                    <li>project 5<\/li>\n                                            \n                                                    <li>new pro0000<\/li>\n                                            \n                                                    <li>pro1212<\/li>\n                                            \n                                                    <li>project 2233<\/li>\n                                            \n                                                    <li>project 255<\/li>\n                                            \n                                                    <li>project 266<\/li>\n                                            \n                                                    <li>project 2277<\/li>\n                                            \n                                                    <li>pro88<\/li>\n                                            \n                                                    <li>new pro 10<\/li>\n                                            \n                                                    <li>new pro11<\/li>\n                                            \n                                                    <li>new pro12<\/li>\n                                            \n                                                    <li>project 13<\/li>\n                                            \n                                                    <li>project 14<\/li>\n                                            \n                                                    <li>asdasd<\/li>\n                                            \n                                                    **<li>pro30<\/li>\n**                                                            <\/ul>\n            <\/li>\n        \n\n\n\n    <\/ul>\n<b>Available Employeees<\/b>\n<ul id=\"myUL2\">\n             <li>unattached  employee<\/li>\n    <\/ul>\n\n","dirty":[]},"serverMemo":{"htmlHash":"5ee28c2b","checksum":"63a3132dafc93f5cc89d19dc5095d74476414b02f90c54ebd33d6f47f3f9a48a"}}`

**Comment by ryatkins** [comment]
Can you post your Blade template?

Do you have wire:key on your navigation elements?
See DOM diffing issues: https://laravel-livewire.com/docs/2.x/troubleshooting
